### PR TITLE
Photo attachment

### PR DESCRIPTION
**Pull request by iiLubos**
image

```
        MMPhotoEditor {
          title: "MMPhotoEditor"
          width: parent.width
          photoUrl: "https://images.pexels.com/photos/615348/forest-fog-sunny-nature-615348.jpeg"

          onTrashClicked: console.log("Move to trash")
          onContentClicked: console.log("Open photo")
        }
```

### CODE REVIEW

File: app/icons/AddImage.svg
Comment (tomasMizera) [inline app/icons/AddImage.svg:1]: We probably want white icon, right?

Comment (iiLubos) [reply app/icons/AddImage.svg:1]: Yes, everywhere :)
Maybe we will change all icons in separate PR

**Comment by PeterPetrik** [comment]
@iiLubos needs to be rebased

File: app/qml/components/MMIcon.qml [hunk 1]
```diff
@@ -23,7 +24,8 @@ Item {
     id: icon
 
     source: __style.arrowLinkRightIcon
-    anchors.centerIn: control
+    anchors.fill: root.useCustomSize ? root : undefined
+    anchors.centerIn: root.useCustomSize ? undefined : root
```
Comment (tomasMizera) [inline app/qml/components/MMIcon.qml:28]: What is the point here? Let's directly create `Image` instead when you need something special :)

File: app/qml/components/MMPhoto.qml [hunk 2]
```diff
@@ -38,19 +38,21 @@ Image {
     }
   }
 
-  MouseArea {
+  Rectangle {
     anchors.fill: parent
-    onClicked: control.clicked(control.photoUrl)
+    color: __style.whiteColor
+    z: -1
```
Comment (tomasMizera) [inline app/qml/components/MMPhoto.qml:44]: What is the reason to use `z`? I highly recommend not using that

File: app/qml/inputs/MMPhotoEditor.qml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+  property url photoUrl
+
+  signal trashClicked()
+
+  contentItemHeight: 160 * __dp
+  spacing: 0
+  radius: 20 * __dp
+
+  content: MMPhoto {
```
Comment (tomasMizera) [inline app/qml/inputs/MMPhotoEditor.qml:30]: Sooo what if there is no photo added yet? :) There is an updated in the design system. We can add it in a different PR though

Comment (iiLubos) [reply app/qml/inputs/MMPhotoEditor.qml:30]: MMPhoto contains already an icon, when photo is not loaded by the design system

File: app/qml/components/MMPhoto.qml (hunk 2 above)
Comment (iiLubos) [reply app/qml/components/MMPhoto.qml:44]: I think, using z here in the base component is safe. This just adds a white background with the 'waiting' icon when the picture is not loaded yet.

File: app/qml/components/MMIcon.qml (hunk 1 above)
Comment (tomasMizera) [reply app/qml/components/MMIcon.qml:28]: https://doc.qt.io/qt-6/qml-qtquick-image.html#fillMode-prop

**Review by tomasMizera — APPROVED** [review]

**Pull request merged**